libavif.pc.cmake

```diff
@@ -1,7 +1,7 @@
 prefix=@CMAKE_INSTALL_PREFIX@
-exec_prefix=${prefix}/bin
-libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
-includedir=${prefix}/include
+exec_prefix=@CMAKE_INSTALL_PREFIX@
```

Comment:
This is equivalent to `exec_prefix=${prefix}`, which you used earlier. I assume you think this is better.

Reply:
Yes, equivalent, and it's a toss-up I guess. We could either go w/ relative way based on ${prefix} for all the entries, or placeholders for all the entries. I don't really have a preference, but as you already commented, this way maybe lets some corner use cases override `CMAKE_INSTALL_FULL_LIBDIR` and `CMAKE_INSTALL_FULL_INCLUDEDIR`...

Reply:
The third option is to use (Autoconf default) placeholders `@prefix@`, `@exec_prefix@`, `@libdir@` and `@includedir@` and associated variables inside the CMakeLists.txt with these defaults (just like libheif and libtiff do as mentioned earlier), which maybe makes tweaking these a smidgeon easier (only one file to be patched).

Reply:
I don't have a preference either. I just wanted to confirm you prefer the new form.

It seems that the variables `prefix` and `exec_prefix` are now defined but not used. Should we delete lines 1-2?

Reply:
Yeah, let's go w/ the PR as is. I think all the lines are potentially used by the pkg-config tool, even if they're not used in the .pc file directly, e.g. there are options to tweak the prefix and I presume the one in the file is the used to remove it from `includedir` for any replacements...

```
     --define-prefix
             Attempts to determine the prefix variable to use for CFLAGS and
             LIBS entry relocations.  This is mainly useful for platforms
             where framework SDKs are relocatable, such as Windows.

     --dont-define-prefix
             Disables the ‘define-prefix’ feature.

     --prefix-variable=VARIABLE
             Sets the ‘prefix’ variable used by the ‘define-prefix’ feature.
```